src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.domain;
+
+public class Car {
+    private String name;
+    private int location;
+
+    public Car(String name, int location) {
+        // 사용자가 잘못된 값을 입력할 경우 `IllegalArgumentException`을 발생시킨 후 애플리케이션은 종료되어야 한다.
+        if (name.length() > 5 || name.isEmpty()) throw new IllegalArgumentException("차 이름은 다섯 글자 이하여야 합니다.");
```

Comment:
예외처리 개행 한 줄 넣으면 좋을 거 같아요! 코드가 조금 길어지네요
```suggestion
        if (name.length() > 5 || name.isEmpty()) 
            throw new IllegalArgumentException("차 이름은 다섯 글자 이하여야 합니다.");
```